src/Arcus.WebApi.Unit/Hosting/TestApiServer.cs

```diff
@@ -71,5 +81,43 @@ protected override IWebHostBuilder CreateWebHostBuilder()
 
             _addServices.Add(services => services.AddScoped(_ => service));
         }
+
+        /// <summary>
+        /// Sets the certificate which the client will use to authenticate itself to this test server.
+        /// </summary>
+        /// <param name="clientCertificate">The client certificate.</param>
+        public void SetClientCertificate(X509Certificate2 clientCertificate)
+        {
+            Guard.NotNull(clientCertificate, nameof(clientCertificate));
+
+            _clientCertificate = clientCertificate;
+        }
+
+        private class CertificateConfiguration : IStartupFilter
```

Comment:
Let's not use inner classes here

Reply:
Not very much use to expose this to the rest of the test project; but no problem for me.

Reply:
It's to align with our other classes, thanks!